opencensus/plugins/grpc/internal/server_filter.cc

```diff
@@ -187,16 +186,13 @@ void CensusServerCallData::Destroy(grpc_call_element *elem,
   double elapsed_time_ms = absl::ToDoubleMilliseconds(elapsed_time_);
   grpc_auth_context_release(auth_context_);
   stats::Record(
-      {{RpcServerErrorCount(),
-        final_info->final_status == GRPC_STATUS_OK ? 0 : 1},
-       {RpcServerRequestBytes(), static_cast<double>(request_size)},
-       {RpcServerResponseBytes(), static_cast<double>(response_size)},
-       {RpcServerServerElapsedTime(), elapsed_time_ms},
-       {RpcServerRequestCount(), sent_message_count_},
-       {RpcServerFinishedCount(), 1},
-       {RpcServerResponseCount(), recv_message_count_}},
-      {{MethodTagKey(), qualified_method_},
-       {StatusTagKey(), StatusCodeToString(final_info->final_status)}});
+      {{RpcServerSentBytesPerRpc(), static_cast<double>(request_size)},
```

Comment:
From the server's point of view, isn't request size = recvd bytes?

Reply:
Good point--switched the variable names.